src/app/routes/utils/fetchPageData/index.test.js

```diff
@@ -13,8 +13,8 @@ import fetchPageData from '.';
 
 const expectedBaseUrl = 'http://localhost';
 const requestedPathname = '/path/to/asset';
-const fullTestPath = 'https://test.bbc.com/hausa/mostwatched.json';
-const fullLivePath = 'https://www.bbc.com/hausa/mostwatched.json';
+const fullTestPath = 'https://test.bbc.com/hausa/mostread.json';
```

Comment:
The urls here don't actually matter if they work on test or live, the test just checks the correct url is passed through. So I could have left it as most watched, but it might confuse future people who see 'most watched' when that is no longer a thing

Reply:
Technically most read paths don't exist either... but for the purposes of this test I suppose it doesn't matter? Or should we consider using a more bff-like path? e.g. https://test.mock-bff.api.bbc.com/simorgh-bff?pageType=bob?

Reply:
I added that one :)